src/main/scala/com/campudus/tableaux/TableauxController.scala

```diff
@@ -28,10 +28,17 @@ class TableauxController(verticle: Verticle) {
     tableaux.create(name)
   }
 
-  def createRow(tableId: Long): Future[DomainObject] = {
-    checkArguments(greaterZero(tableId))
-    verticle.logger.info(s"createRow $tableId")
-    tableaux.addRow(tableId)
+  def createRow(tableId: Long, values: Option[Seq[(Long, _)]]): Future[DomainObject] = {
+    values match {
+      case Some(v) =>
+        checkArguments(greaterZero(tableId), notNull(v))

```

Comment:
You should not need to check for null, that's why you have used an option here, didn't you?